Skulls/build.gradle

```diff
@@ -0,0 +1,75 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+group 'com.jme3'
+version '1.0'
+
+mainClassName = "com.jme3.skulls.MainApplication"
+
+repositories {
+    mavenCentral()
+    jcenter()
```

Comment:
This should at least be a deprecation warning if not straight not working, right? Or is jcenter still in this read-only archived state?